docs/docs/themes/shadowing.md

```diff
@@ -0,0 +1,215 @@
+---
+title: Shadowing
+---
+
+
+Gatsby themes introduce a concept called "shadowing". This feature allows users to override a component, or any other file, in the `src` directory to customize it.
+
+A practical use case is when you've installed `gatsby-theme-blog` and want to customize the author Bio component to add your own biographical content. Component shadowing lets you replace the theme’s original file, `gatsby-theme-blog/src/components/bio.js`, with your own file to override with any changes you need.
```

Comment:
We should probably just drop `Component` at this point. Also added code ticks around the `Bio` name literal. While shadowing is commonly used to override, it can also be used to extend components or massage data as it flows through the site, so I changed `override` to `make`.

```suggestion
A practical use case is when you've installed `gatsby-theme-blog` and want to customize the author `Bio` component to add your own biographical content. Shadowing lets you replace the theme’s original file, `gatsby-theme-blog/src/components/bio.js`, with your own file to make any changes you need.
```